src/Goteo/Model/Node.php

```diff
@@ -72,6 +71,11 @@ class Node extends \Goteo\Core\Model {
         $config
         ;
 
+    /**
+     * @deprecated
+     */
+    public ?string $iframe;
```

Comment:
@javicarrillo @davidbeig If we don't need this property anymore, could we remove the related DB column perhaps and then the property?